es/utils/crypto.js

```diff
@@ -23,10 +23,10 @@
 
 import bs58check from 'bs58check'
 import RLP from 'rlp'
-import {blake2b} from 'blakejs'
+import { blake2b } from 'blakejs'
```

Comment:
Is this really necessary? Will you change to `{ X }` style everywhere else in the project so it's at least coherent?